### PR TITLE
Provides RELEASE.md file with the release instructions.

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### What does this PR do?

Provides RELEASE.md file with the release instructions.

### CODE REVIEW

**Comment by ibuziuk** [comment]
@vparfonov  what about the bug-fix / service release process? are you going to provide this info in a separate PR ?

**Comment by vparfonov** [comment]
> @vparfonov what about the bug-fix / service release process? are you going to provide this info in a separate PR ?

Good catch, I will add here

**Comment by ibuziuk** [comment]
@vparfonov does che-parent / che-docs update suppose to be covered by this RELEASE.md?

**Comment by nickboldt** [comment]
OH NO! Happy path tests are failing for a new, document-only file. 

Why are the happy paths failing for a file that isn't part of the build?

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  * on MacOS: `set_tag_version_images_macos.sh {tag}`
+  * `git commit` 
+  * `git push`
+##### 4. Start pre-release testing.
+##### 5. If pre-release test passed need to merge branch to the `release` branch and push changes, release process will start by webhook:
+* `git checkout release`
+* `git merge -X theirs {branchname}`
+* `git push -f`
+##### 6. Close/release repository on Nexus 
+ https://oss.sonatype.org/#stagingRepositories
```
Comment (nickboldt) [inline RELEASE.md:30]: Can this be scripted? Who has login/nexus credentials to make such a change?

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    2 Update default version of plugins and editors in che.properties to released tag e.g `7.7.0`
+    3 Update Helm charts with released tag e.g `7.7.0`
+    
+  To do this execute commands:
+  * `cd .ci`
+  * on Linux machine: `set_tag_version_images_linux.sh {tag}` 
+  * on MacOS: `set_tag_version_images_macos.sh {tag}`
+  * `git commit` 
+  * `git push`
+##### 4. Start pre-release testing.
```
Comment (nickboldt) [inline RELEASE.md:24]: Link to job? trigger instructions? Who to talk to? deadline for review? Criteria for rejection/approval?

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* Update parent version : `mvn versions:update-parent  versions:commit -DallowSnapshots=true -DparentVersion={next_version}`
+* Update dependencies: `sed -i -e "s#{version_old}#{next_version}#" pom.xml`
+* `git commit`
+* `git push --set-upstream origin set_next_version_in_master_{next_version}`
+* Create PR
+##### 3. In release branch of Che need to set up released version of plugin registry and plugins:
+    1 Update deploy_che.sh (should be deprecated soon - https://github.com/eclipse/che/issues/14069) default environment variables
+    2 Update default version of plugins and editors in che.properties to released tag e.g `7.7.0`
+    3 Update Helm charts with released tag e.g `7.7.0`
+
```
Comment (nickboldt) [inline RELEASE.md:17]: Could you list actual files that are changed or provide a link to PRs for 7.7.0 and 7.7.1?

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+##### 1. Create branch for release preparation and next bugfixes:
+* `git branch {branchname} #e.g 7.7.x`
+* `git push --set-upstream origin {branchname}`
+##### 2. Create PR for switch master to the next development version :
+* `git branch set_next_version_in_master_{next_version} #e.g 7.8.0-SNAPSHOT`
+* Update parent version : `mvn versions:update-parent  versions:commit -DallowSnapshots=true -DparentVersion={next_version}`
+* Update dependencies: `sed -i -e "s#{version_old}#{next_version}#" pom.xml`
+* `git commit`
+* `git push --set-upstream origin set_next_version_in_master_{next_version}`
+* Create PR
```
Comment (nickboldt) [inline RELEASE.md:12]: link to samples please. See https://github.com/eclipse/che-machine-exec/blob/master/RELEASE.md for examples of how to do this.

File: RELEASE.md (hunk 2 above)
Comment (nickboldt) [reply RELEASE.md:24]: See https://github.com/eclipse/che-machine-exec/blob/master/RELEASE.md for example of how to link to the job(s) that run the builds/tests so we an immediate see status.

**Review by nickboldt — CHANGES_REQUESTED** [review]
Good start, but needs fleshing out with more details, links, samples, etc.

**Comment by nickboldt** [comment]
Is this RELEASE.md for all three Che projects (parent, docs, che) ? What about che-dev or che-lib? Will you also create RELEASE.md files for those repos? 

Or does the Che release process handle all 4 projects in one operation?

**Review by nickboldt — APPROVED** [review]
Unblocking so we can merge; remaining items to be done as enhancement as part of the doc generated from tomorrow's collab release with @mkuznyetsov and @vparfonov

**Pull request merged**